src/main/resources/resources/logisim/strings/gui/gui.properties

```diff
@@ -621,6 +621,26 @@ windowGridColorsReset = Reset grid colors to Logisim's defaults
 windowSetAutoScaleFactor = Set zoom factor to Logisim's default
 windowCanvasLocation = Main canvas location:
 #
+# prefs/HotkeyOptions.java
+#
+hotkeyOptHeader = This Tab allows users to dynamically change the shortcut keys.Note that the hotkeys on the menubar need to be started with CTRL.
+hotkeyOptHelp = To change the key, simply click the button right-side. Then directly press the keys you want, and press OK to save the result.
+hotkeyOptTitle = Hotkey Settings
+hotkeyOptResetBtn = Reset to Default
+hotkeySimAutoPropagate = Auto-Propagate
+hotkeySimReset = Reset Simulation
+hotkeySimStep = Single-Step Propagation
+hotkeySimTickHalf = Manual Tick Half Cycle
+hotkeySimTickFull = Manual Tick Full Cycle
+hotkeySimTickEnabled = Auto-Tick Enabled
+hotkeyEditUndo = Edit Undo
+hotkeyEditRedo = Edit Redo
+hotkeyFileExport = Export File
+hotkeyFilePrint = Print File
+hotkeyFileQuit = Quit Logisim
+hotkeyErrCtrl = This key combo must start with CTRL.
+hotkeyErrConflict = This key is comflicted with :
+#
```

Comment:
Please add these keys like `# ==> hotkeyOptHeader =` in all the gui_*.properties files. This helps the person who is going to translate which keys are still not available in the given language.

Reply:
done